ingestion/src/metadata/cmd.py

```diff
@@ -123,4 +131,80 @@ def report(config: str) -> None:
         ) from exc
 
 
+@metadata.command()
+@click.option(
+    "-t",
+    "--type",
+    help="Workflow config",
+    required=True,
+)
+@click.option(
+    "-p",
+    "--path",
+    help="Workflow config",
+    required=False,
+)
+def docker_run(type: str, path: str = "") -> None:
+    """
+    Checks Docker Memory Allocation
+    Run Local Docker - metadata docker-run -t local
+    Run Latest Release Docker - metadata docker-run -t latest
+    """
+    try:
+        client = docker.from_env()
+        docker_info = client.info()
+        calc_gb = 1024 * 1024 * 1000
+        if docker_info["MemTotal"] < (3 * 1024 * 1024 * 1000):
+            raise MemoryError(
+                f"Please Allocate More memory to Docker.\nRecommended: 4GB\nCurrent: "
+                f"{round(float(docker_info['MemTotal']) / calc_gb, 2)}"
+            )
+        if type == "local":
+            logger.info("Running Local Docker")
+            if path == "":
+                raise ValueError("Please Provide Path to local docker-compose.yml file")
+            else:
```

Comment:
Else is not required since the above if raises an error, exiting this function.

```
if path == "":
     raise ValueError("Please Provide Path to local docker-compose.yml file")
subprocess.run(  
```

Reply:
Made the suggested changes in the recent commit